spec/support/pages/projects/show.rb

```diff
@@ -56,7 +54,12 @@ def expect_no_visible_sidebar
         expect(page).to have_no_css(".op-grid-page--grid-container")
       end
 
-      def within_async_loaded_sidebar(&)
+      def expect_visible_sidebar
+        expect_angular_frontend_initialized
+        expect(page).to have_css(".op-grid-page--grid-container")
```

Comment:
I think the CSS class here is wrong 😁 shouldn't it rather be:

```suggestion
        expect(page).to have_css(".op-grid-page--sidebar")
```